homeassistant/components/sql/config_flow.py

```diff
@@ -23,6 +23,9 @@
 
 DATA_SCHEMA = vol.Schema(
     {
+        vol.Required(CONF_NAME, default="Select SQL Query"): selector.TextSelector(
+            selector.TextSelectorConfig()
+        ),
```

Comment:
This works too:
```suggestion
        vol.Required(CONF_NAME, default="Select SQL Query"): str,
```

Reply:
Not if you want to use the data_description according to documentation